.github/workflows/test-e2e-cypress.yml

```diff
@@ -0,0 +1,145 @@
+        default: "8.1"
+    secrets:
+      record-key:
+        description: 'Record key for Cypress Dashboard'
+        required: false
+
+jobs:
+  test_cypress_e2e:
+    name: E2E Test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        spec:
+          - accordion.cypress.js
+          - alert.cypress.js
+          - click-to-tweet.cypress.js
+          - counter.cypress.js
+          - dynamic-separator.cypress.js
+          - event-item.cypress.js
+          - events.cypress.js
+          - faq.cypress.js
+          - feature.cypress.js
+          - features.cypress.js
+          - food-item.cypress.js
+          - food-and-drinks.cypress.js
+          - form.cypress.js
+          - gallery-carousel.cypress.js
+          - gallery-collage.cypress.js
+          - gallery-masonry.cypress.js
+          - gallery-offset.cypress.js
+          - gallery-stacked.cypress.js
+          - gif.cypress.js
+          - gist.cypress.js
+          - hero.cypress.js
+          - highlight.cypress.js
+          - icon.cypress.js
+          - logos.cypress.js
+          - map.cypress.js
+          - media-card.cypress.js
+          - opentable.cypress.js
+          - post-carousel.cypress.js
+          - posts.cypress.js
+          - pricing-table-item.cypress.js
+          - pricing-table.cypress.js
+          - column.cypress.js
+          - row.cypress.js
+          - service.cypress.js
+          - services.cypress.js
+          - shape-divider.cypress.js
+          - social-profiles.cypress.js
+          - modal.cypress.js
+          - media-filter-control.cypress.js
+          - settings-modal-control.cypress.js
+          - coblocks-labs.cypress.js
+          - settings-modal-control.cypress.js
+          - layout-selector.cypress.js
+          - lightbox-controls-gallery.cypress.js
+          - lightbox-controls-image.cypress.js
+          - media-text-styles.cypress.js
+          - padding-controls.cypress.js
+          - settings-modal-control.cypress.js
+
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup wp-env
+        uses: godaddy-wordpress/setup-wp-env@v1
+        with:
+          core: ${{ inputs.wpVersion }}
+          phpVersion: ${{ inputs.phpVersion }}
+          plugins: '["."]'
+          themes: '["https://downloads.wordpress.org/theme/go.zip"]'
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
+
+      - name: Setup WP-CLI
+        uses: godaddy-wordpress/setup-wp-cli@1
+
+      - name: Install dependencies
+        run: |
+          composer install --prefer-dist --optimize-autoloader &
+          yarn install --immutable
+
+      - name: Build plugin
+        run: |
+          yarn build
+
+      - name: Prepare Go
+        if: ${{ inputs.theme == 'go' }}
+        run: |
+          cd $(wp-env install-path)/go
+          mkdir -p coblocks/icons
```

Comment:
This custom SVG icon was not Go-specific, but CoBlocks specific (the Icon block), so it should be executed for any theme.

Reply:
We may need to circle back to this, then. As we will need to find an alternate way to use WP-ENV and add these into any theme within tests.

Reply:
Ok, but just to give a little context, we added this last year because we realized that when we added a custom icon (like specified here : https://github.com/godaddy-wordpress/coblocks/blob/master/docs/hooks/icons-block.md ) it would break the Icon block. So, the idea is just to make sure that we don't break that functionality again in CoBlocks.

Reply:
Okay, I've got this covered in the latest change: https://github.com/godaddy-wordpress/coblocks/pull/2451/commits/1bed6e1816396589a47363b11ca6fda3c45b5539